AppAuth-JS-Ext-Logout/built/end_session_request_test.js

```diff
@@ -1,49 +0,0 @@
-"use strict";
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file